react/components/FloatingActionBar/index.tsx

```diff
@@ -30,7 +30,7 @@ const FloatingActionBar: FC<PropTypes.InferProps<typeof propTypes>> = ({
   save: { label: saveLabel, ...saveProps },
   cancel: { label: cancelLabel, ...cancelProps },
 }) => (
-  <div className="styleguide__floating-action-bar shadow-active w-100 bg-base tr pv5 pr7 absolute fixed bottom-0 left-0 z-2">
+  <div className="styleguide__floating-action-bar shadow-active w-100 bg-base tr pv5 pr7 absolute fixed bottom-0 left-0 z-999">
```

Comment:
Is this not going to conflict with the modal?

Reply:
I don't think so, the modal z-index is 10000